packages/eslint-plugin-jest-playwright/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  configs: {
+    recommended: {
+      plugins: ['playwright/jest-playwright']
```

Comment:
I believe we want the recommended config to extend this since that is what it did before.
```suggestion
      extends: 'plugin:playwright/jest-playwright'
```